### PR TITLE
Add example notebook

### PR DESCRIPTION
**Pull request by meanderingstream**
Created an example notebook that allows a user to upload an image file.  Demonstrates center cropping the image. Also demonstrates resizing the image.  Provides some narrative on why resizing and normalizing an image might be useful for
neural network training.

### CODE REVIEW

File: notebooks/nx_image_demo.livemd [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# Nx.Image Demonstration
+
+```elixir
+Mix.install([
+  {:nx, "~> 0.4.0"},
+  {:kino_vega_lite, "~> 0.1.6"},
+  {:kino, "~> 0.7.0"},
+  {:nx_image, git: "https://github.com/elixir-nx/nx_image.git"},
+  {:stb_image, "~> 0.5.2"},
+  {:vega_lite, "~> 0.1.6"},
```
Comment (josevalim) [inline notebooks/nx_image_demo.livemd:10]: I think we can let some of the transitive dependencies to their job. :)

```suggestion
  {:kino_vega_lite, "~> 0.1.6"},
  {:nx_image, git: "https://github.com/elixir-nx/nx_image.git"},
  {:stb_image, "~> 0.5.2"},
```

File: notebooks/nx_image_demo.livemd [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  {:file_upload_smart_cell, git: "https://github.com/akoutmos/file_upload_smart_cell.git"}
+])
+```
+
+## Upload your test image
```
Comment (josevalim) [inline notebooks/nx_image_demo.livemd]: The Kino main branch has a `Kino.Input.image/1`, which supports file uploads and, best of all, returns the image in RGB / Tensor format input. Could you please update this section to use that instead and remove the `file_upload_smart_cell` dependency for now? :)

https://github.com/livebook-dev/kino/blob/5a015e6b9f2097452180457d1c39277b4b893295/lib/kino/input.ex#LL262C17-L262C22

File: notebooks/nx_image_demo.livemd [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+```
+
+## Visualizing the Resized Image
+
+```elixir
+resized_image =
+  resized_tensors
+  |> StbImage.from_nx()
+  |> StbImage.to_binary(:png)
+  |> Kino.Image.new(:png)
```
Comment (josevalim) [inline notebooks/nx_image_demo.livemd]: Kino main also allows a tensor to be given directly to `Kino.Image`, which allows us to get rid of StbImage altogether. :)

**Review by josevalim — COMMENTED** [review]
This looks great @meanderingstream! 

It just happened that we added some new features in Kino to make working with images easier (literally this week!), so could you point to Kino main and give them a try? :)

File: notebooks/nx_image_demo.livemd (hunk 2 above)
Comment (meanderingstream) [reply notebooks/nx_image_demo.livemd]: I figured out that I needed to use the latest Livebook code.  I succeeded in adding the Kino.Input.image("label) to a notebook.  However, I'm having difficulty figuring out how to get any data about the image (size or tensor).  

img = Kino.Input.image("label")

IO.inspect(img)
-->
%Kino.Input{
  attrs: %{
    default: nil,
    destination: {Kino.SubscriptionManager, :"dceq47pv-livebook_iaakhca6@ml3"},
    fit: :contain,
    format: :rgb,
    id: "83729853",
    label: "label",
    ref: "l266kaqz5xj753jhfry5i2sp76cfdnsy",
    size: nil,
    type: :image
  }
}

Comment (meanderingstream) [reply notebooks/nx_image_demo.livemd]: I figured out that I need to read the input.

Kino.Input.read(img)

Comment (meanderingstream) [reply notebooks/nx_image_demo.livemd]: Switched to Kino.Input.image and removed StgImage.

**Review by jonatanklosko — APPROVED** [review]
@meanderingstream thanks! I just pushed some edits and added the notebook to the docs :)

**Pull request merged**